manifests/init.pp

```diff
@@ -4,10 +4,14 @@
 #
 # @see http://www.freedesktop.org/software/polkit/docs/latest/ PolicyKit Documentation
 #
+# @param package_ensure The ensure status of packages
+#
 # @author Trevor Vaughan <mailto:tvaughan@onyxpoint.com>
 #
-class polkit {
+class polkit (
+  Polkit::PackageEnsure $package_ensure = 'installed'
```

Comment:
This needs to be tied to a `simp_options` catalyst. We *really* don't want people having to hit every module if they want to change the default nature of the system and it would be nice not to have to do this twice everywhere.

@jeefberkey Can you run this by the mailing list and make sure that everyone is OK with a `simp_options::package_update_strategy` (or something) parameter and then do that and tie these together?

Otherwise, the code is fine.